src/components/Common/FilePreviewDialog.tsx

```
@@ -170,6 +183,67 @@
     onClose?.();
   };
 
+  const handleRotate = (rotation: number) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'handleRotate'. Consider to delete it or rename it.

'handleRotate' is defined here:


(lint/suspicious/noRedeclare)